dev.to/dev.io/devto.js

```diff
@@ -6,6 +6,30 @@ var URL = "https://dev.to/top/week";
 
 let c=0;
 const rp = require('request-promise');
+const imagearray= [
```

Comment:
Can we have it dynamically extracted instead of the array? Every article has its own header image🤔

Reply:
@shubhank-saxena sir unline medium,  dev.to does not display the header images in the list of blogs... I have to open each blog individually and scrape the images which will increase the execution time of the program to a great extend as instead of 1 request it would then have to handle 10-12 requests per execution if you want I can still do it, slack events-API give us only 3s to complete the request, so sometimes the execution might fail

Reply:
Ohh🤔

Reply:
sir, I am finding a soln. to this problem from the past few days since I deployed the bot as soon as I find one I will update the code. For the time being this was only left that I could think of